docs/reference/query-dsl/boosting-query.asciidoc

```diff
@@ -40,14 +40,15 @@ GET /_search
 [[boosting-top-level-params]]
 ==== Top-level parameters for `boosting`
 
-`positive` (Required)::
-Query you wish to run. Any returned documents must match this query.
+`positive`::
+(Required, query object) Query you wish to run. Any returned documents must
+match this query.
 
-`negative` (Required)::
+`negative`::
 +
 --
-Query used to decrease the <<query-filter-context, relevance score>> of matching
-documents.
+(Required, query object) Query used to decrease the <<query-filter-context,
```

Comment:
I don't mean to nitpick too much, but I wonder if 'Required' and 'Optional' should be lowercase to match the types like 'query object' and 'string'?

Reply:
I'm going to leave this capitalized for now so it's consistent with the [Elastic API Reference template](https://raw.githubusercontent.com/elastic/docs/master/shared/api-ref-ex.asciidoc). I can easily change this later if we decide to change the capitalization in the template.